app/views/shared/_footer_lite.html.erb

```diff
@@ -14,7 +14,7 @@
       </div>
     </div>
     <div class='i18n-mobile-dropdown tablet:display-none display-none'>
-      <ul class='usa-list--unstyled margin-bottom-0 text-white text-center'>
```

Comment:
`text-white` had no meaningful effect here, since default link styling would override the inherited value.